analyzer/src/analyze/attribute.py

```diff
@@ -48,7 +49,9 @@ def cast_type(self, value):
         try:
             return self.normalizer(value)
         except Exception as e:
-            logger.error(
+            logger.warning(
```

Comment:
In the case we don't have a condition to manage that,
What will happen if we try to add, for example, a not-castable string in an invalid fhir type, like decimal ?

There will only be a warning and the attribute will not be mapped ?

Reply:
https://github.com/arkhn/fhir-river/issues/155

Reply:
There'll be a warning logged during the analyses and then most probably a validation error in the loader

Reply:
According to me, it's perfect 🙂